test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/CheckContactDetailsMrnControllerSpec.scala

```diff
@@ -187,26 +186,14 @@ class CheckContactDetailsMrnControllerSpec
 
         inSequence {
           mockAuthWithNoRetrievals()
-          mockGetSession(session.copy(journeyStatus = Some(fillingOutClaim)))
+          mockGetSession(session)
         }
 
-        checkPageIsDisplayed(
+        checkIsRedirect(
```

Comment:
could we get a test for the new controller method in the claimant-details controller please